### PR TITLE
Added: Faker.Address.Es and Faker.Address.En

### PR DESCRIPTION
**Pull request by vbrazo**
I've added:

- [x] USAGE.md docs if applicable
- [x] CHANGELOG.md
- [x] Faker.Address.Es and its doctests
- [x] Faker.Address.En and its doctests
- [x] Faker.Address doctests

### CODE REVIEW

File: lib/faker/address/es.ex [hunk 1]
```diff
@@ -0,0 +1,734 @@
...
+
+  @moduledoc """
+  Functions for generating addresses in Spanish
+  """
+
+  @doc """
+  Return random building number.
+  """
+  @spec building_number() :: String.t()
+  def building_number do
```
Comment (igas) [inline lib/faker/address/es.ex]: if the function is not different from main module `Faker.Address` I don't think we need to override it. The idea is to always use main modules without locale names in the name, while config will determinate which one to call.

File: lib/faker/address/es.ex [hunk 2]
```diff
@@ -0,0 +1,734 @@
...
+    else
+        geo_to_bits(n, position - 2, {gmin, mid})
+    end
+  end
+
+  @doc """
+  Return random latitude.
+  """
+  @spec latitude() :: float()
+  def latitude do
```
Comment (igas) [inline lib/faker/address/es.ex]: same thing here, I really hope that `latitude` is the same thing in all countries :)

Will not comment in other functions like this, please check them too.

File: lib/faker/address/es.ex [hunk 3]
```diff
@@ -0,0 +1,734 @@
...
+    "America/Caracas",
+    "America/La_Paz",
+    "America/Santiago",
+    "America/St_Johns",
+    "America/Sao_Paulo",
+    "America/Argentina/Buenos_Aires",
+    "America/Guyana",
+    "America/Godthab",
+    "Atlantic/South_Georgia",
+    "Atlantic/Azores, Atlantic/Cape_Verde",
```
Comment (igas) [inline lib/faker/address/es.ex]: typo, should be 2 strings

File: test/faker/address/es_test.exs [hunk 4]
```diff
@@ -0,0 +1,88 @@
+defmodule Faker.Address.EsTest do
+  use ExUnit.Case, async: true
+
+  test "building_number/0" do
+    assert is_binary(Faker.Address.Es.building_number)
+    assert String.length(Faker.Address.Es.building_number) in 1..5
```
Comment (igas) [inline test/faker/address/es_test.exs]: this kind of tests make sense, but should be like the one in `color_test.exs` for example. While is_binary should go to doctests.

**Comment by igas** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: lib/faker/address/es.ex (hunk 1 above)
Comment (vbrazo) [reply lib/faker/address/es.ex]: I see your point 👍

File: test/faker/address_test.exs [hunk 5]
```diff
@@ -1,6 +1,9 @@
 defmodule AddressTest do
   use ExUnit.Case, async: true
 
+  doctest Faker.Address.En
+  doctest Faker.Address.Es
+
   test "building_number/0" do
     assert is_binary(Faker.Address.building_number)
     assert String.length(Faker.Address.building_number) in 1..5
```
Comment (vbrazo) [inline test/faker/address_test.exs]: I also have to convert these binary tests to doctests. I'll do that tomorrow.

File: lib/faker/address/es.ex [hunk 6]
```diff
@@ -525,6 +525,10 @@ defmodule Faker.Address.Es do
       "Elizabeth Mercado 26 Esc. 610"
       iex> Faker.Address.Es.street_address(false)
       "Ryan Enrique Sector , 05"
+      iex> Faker.Address.Es.street_address(false)
+      "Price Colonia , 02"
```
Comment (vbrazo) [inline lib/faker/address/es.ex]: This space close to the comma seems pretty weird to me. Let me fix that.

**Pull request merged**